src/components/mortgage-investment-compare/amortization-table.js

```diff
@@ -12,7 +12,7 @@ const tableDivider = css`
 `
 const fixedTableLayout = css`
   display: table;
-  width: 100%;
+  width: 53.5rem;
```

Comment:
I had to give the table a fixed width. Let me know how it looks on your end. This should solve the issue with the thead and the tbody being misaligned but it has a secondary minor side effect which I prefer: the scrollbar will be separated from the tbody when you zoom out. We'll come up with a better implementation of this table some time in the future but for now I think this works well